colour/difference/delta_e.py

```diff
@@ -188,11 +188,11 @@ def delta_E_CIE1994(Lab_1, Lab_2, textiles=False):
     delta_A = a_1 - a_2
     delta_B = b_1 - b_2
 
-    delta_H = np.sqrt(delta_A ** 2 + delta_B ** 2 - delta_C ** 2)
+    delta_H = delta_A ** 2 + delta_B ** 2 - delta_C ** 2
 
     L = (delta_L / (k_L * s_L)) ** 2
     C = (delta_C / (k_C * s_C)) ** 2
-    H = (delta_H / (k_H * s_H)) ** 2
+    H = (delta_H / (k_H * s_H)** 2) 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/app/colour-science/colour/pullRequest?prid=4714911)

Reply:
fixed in next commit